### DISCUSSION TITLE
Translating translatable_method like "Create a New Account"

### DISCUSSION THREAD

**Discussion by learn-ai763** [janko/rodauth-rails#329] [General]
I wanted to translate the dynamic strings in `views/rodauth/_login_form_footer.html.erb`

```ruby
<% rodauth.login_form_footer_links.sort.each do |_, link, text| %>
  <li><%= link_to text, link %></li>
<% end %>
```

These are depending on what features you have turned on. So I need to look up the feature definitions. As an example I had verify_account turned on, so in its feature file eg:  `<gem_path>/features/verify_account.rb`

found this line
`translatable_method :verify_account_resend_link_text, "Resend Verify Account Information"`

The only thing I need to do is to add this to `rodauth_main.rb`, so I add a translation new section at the end of `configure` eg:

```ruby
class RodauthMain < Rodauth::Rails::Auth
  configue do
    ... lots of things ...
  
    # ==> Translations
   verify_account_resend_link_text { I18n.t('resend_verify_account_info') }
  end
end
```

Maybe this makes this whole thing unnecessary: https://github.com/janko/rodauth-i18n

**Comment by janko** [comment]
Note sure what the question/suggestion is. rodauth-i18n is definitely convenient because it automates this process, so you can just edit YAML files directly. The `rodauth:i18n` generator also automatically imports all translatable configuration.
